quinn-proto/src/config.rs

```diff
@@ -299,6 +301,7 @@ where
     /// Create a cid generator for local cid in Endpoint struct
     pub(crate) connection_id_generator_factory:
         Arc<dyn Fn() -> Box<dyn ConnectionIdGenerator> + Send + Sync>,
+    pub(crate) supported_versions: RangeInclusive<u32>,
```

Comment:
Hmm, not sure a range is expressive enough. Maybe just `Vec<u32>` or `&'static [u32]`?